src/pages/components/CertificateForm.js

```diff
@@ -7,20 +7,22 @@ import CertificateInputFields from './CertificateInputFields'
 import CertificateFormHeader from './CertificateFormHeader'
 import CertificateActionsButtons from './CertificateActionsButtons'
 
-export default function CertificateForm() {
+const useCallbackChVal = (set) => useCallback((e) => set(e.target.value), [set])
+
+export default function CertificateForm(props) {
   const [sdVal, setSdVal] = useState('2024-01-01')
   const [stVal, setStVal] = useState('00:00')
   const [edVal, setEdVal] = useState('2024-01-01')
   const [etVal, setEtVal] = useState('23:55')
   const [enVal, setEnVal] = useState('')
   const [szVal, setSzVal] = useState('')
 
-  const handleSdChgeVal = (e) => setSdVal(e.target.value)
-  const handleStChgeVal = (e) => setStVal(e.target.value)
-  const handleEdChgeVal = (e) => setEdVal(e.target.value)
-  const handleEtChgeVal = (e) => setEtVal(e.target.value)
-  const handleEnChgeVal = (e) => setEnVal(e.target.value)
-  const handleSzChgeVal = (e) => setSzVal(e.target.value)
+  const handleSdChgeVal = useCallbackChVal(setSdVal)
+  const handleStChgeVal = useCallbackChVal(setStVal)
+  const handleEdChgeVal = useCallbackChVal(setEdVal)
+  const handleEtChgeVal = useCallbackChVal(setEtVal)
+  const handleEnChgeVal = useCallbackChVal(setEnVal)
+  const handleSzChgeVal = useCallbackChVal(setSzVal)
 
   const handleSubmit = (event) => {
```

Comment:
Missed this one for a `useCallback`. I think this can be moved outside the component in entirety for now

Reply:
I think I've missread this .. thought you are reffering to the line above . So , add a useCallback wrapper function around handleSubmit ?!

> I think this can be moved outside the component in entirety for now

I guess partially not entirety ? Anyway .. i guess this is good enough since this will change..